arch/common/ua_freeRTOS.h

```diff
@@ -30,7 +28,11 @@
 # define UA_realloc realloc
 #endif
 
-#define UA_access access
+#ifdef UA_ENABLE_DISCOVERY
```

Comment:
`UA_fileExists` is only necessary if `UA_ENABLE_DISCOVERY_SEMAPHORE` is enabled.

It's better to change these lines to:

```c
#ifdef UA_ENABLE_DISCOVERY_SEMAPHORE
#error freeRTOS does not support file handling. Disable UA_ENABLE_DISCOVERY_SEMAPHORE
#endif
```

Reply:
You're right, I'll change UA_ENABLE_DISCOVERY to UA_ENABLE_DISCOVERY_SEMAPHORE. But I don't it's always an error. Maybe you have a file manager code that can do this function, so you could use this and define it before including the fie